### ISSUE TITLE
AB - Fix photoyear shapefile

### ISSUE THREAD

**Issue by MelinaHoule** [CASFRI/CASFRI#766]
Alberta used a shapefile representing inventory aerial photograph acquisition to assign photo year at the polygon level. The shapefile was acquired from the province in 2009. For the Al-Pac's FMAs, we received two independant PDFs that has a better information about the year of photograph acquisition within the FMA's.

We need to update the shapefile for the Al-Pacs FMAs.

**Comment by MelinaHoule** [comment]
Alpac aerial photograph acquisition PDFs were added in FRIs/AB/AB14/doc/map/. 

1) AVI - Photo Hixtory Map - 92-2001.pdf
2) savi_photo_plan_apr1813.pdf

**Comment by MelinaHoule** [comment]
For Alpac FMA, we received from a collaborator a table that links township to original AVI photo acquisition (1991-2001).  Using a generic township shapefile from Alberta, we recreated the photoyear shapefile. We used the 2 pdf acquired above and the Photoyear_update.shp (used in the rest of Alberta) to fill the blank. All inventories associated with Alpac AVI should use this shapefile. Alpac inventory post AVI (such as SAVI) should use the savi_photoyear shapefile.
